koji-blame

```diff
@@ -68,7 +69,7 @@ def parse_history(lth_output, since=EPOCH):
                 user = "Edgar Fajardo"
             user = re.sub(r" A?\d+$", "", user)
             date = strptime(m.group('date'), "%a %b %d %H:%M:%S %Y")
-            if date > since:
+            if date > since and date < until:
```

Comment:
I think you meant `since < date < until` - but pls don't do that either.

Or is there some reason to exclude items with a future timestamp?

Unless there is, i suggest something more like `since < date < until if until else since < date` -- or equivalent